### PR TITLE
Trying to fix issue #384

### PR DESCRIPTION
**Pull request by schaffer**
Implemented floatlabels for input fields when composing email.

### CODE REVIEW

**Comment by dudadornelles** [comment]
Hey, I think it would be nice if you could move the "function floatlabelHandler(element)" into a flightjs mixin: https://github.com/flightjs/flight/blob/master/doc/mixin_api.md

File: web-ui/app/js/mail_view/ui/compose_box.js [hunk 1]
```diff
@@ -57,6 +57,26 @@ define(
         this.select('recipientsFields').show();
         this.on(this.select('closeButton'), 'click', this.showNoMessageSelected);
         this.enableAutoSave();
+
+        function floatlabelHandler(element) {
+          $(function() {
```
Comment (dudadornelles) [inline web-ui/app/js/mail_view/ui/compose_box.js]: You probably dont need this to be on document.ready.

File: web-ui/app/templates/compose/compose_box.hbs [hunk 2]
```diff
@@ -1,8 +1,14 @@
 <button class="close-mail-button">
   <i class="fa fa-times"></i>
 </button>
-<input type="text" id="subject" value="{{subject}}" placeholder="{{t 'Subject'}}" tabindex="1"/>
-<textarea id="text-box" placeholder="{{t 'Body'}}" tabindex="2">{{body}}</textarea>
+<div class="compose">
```
Comment (dudadornelles) [inline web-ui/app/templates/compose/compose_box.hbs]: I'm not sure about the name "compose" for the class. In this case you won't need any class actually given it is the only div inside the whole thing

Comment (rafaelzlisboa) [inline web-ui/app/templates/compose/compose_box.hbs]: @dudadornelles i think it isn't, there's also `buttons-group columns` and a `clearfix` down there \/\/\/

File: web-ui/app/js/mail_view/ui/draft_box.js [hunk 3]
```diff
@@ -72,6 +72,26 @@ define(
         this.enableAutoSave();
         this.on(this.select('cancelButton'), 'click', this.showNoMessageSelected);
         this.on(this.select('closeMailButton'), 'click', this.showNoMessageSelected);
+
+        function floatlabelHandler(element) {
```
Comment (rafaelzlisboa) [inline web-ui/app/js/mail_view/ui/draft_box.js]: same function declared twice is not very nice :(
would it be possible to use the [mixins feature](https://github.com/flightjs/flight/blob/master/doc/mixin_api.md) from flightjs?

File: web-ui/app/templates/compose/compose_box.hbs (hunk 2 above)
Comment (schaffer) [inline web-ui/app/templates/compose/compose_box.hbs]: There is css for this div. What name should I use? Maybe 'floatlabel' as well?

**Comment by schaffer** [comment]
Hey guys, is everything ok now?

**Comment by jeffhsta** [comment]
Hey @schaffer I saw your pull request and it's working, but I am just missing the JS tests.
Could you create them?

**Pull request merged**